### ISSUE TITLE
Discrepancy in ILAENV behavior

### ISSUE THREAD

**Issue by youyu3** [Reference-LAPACK/lapack#479]
There are two ILAENV defined in the code base, `SRC/ilaenv.f `and `INSTALL/tstiee.f`.  And the logic below is different in the two definitions.  Is there a bug in one of them?

BTW, the documentation here does describe what IEEECK does either.

Thanks.

`SRC/ilaenv.f`:

```
  140 CONTINUE
*
*     ISPEC = 10: ieee NaN arithmetic can be trusted not to trap
*
*     ILAENV = 0
      ILAENV = 1
      IF( ILAENV.EQ.1 ) THEN
         ILAENV = IEEECK( **1**, 0.0, 1.0 )
      END IF
      RETURN
*
  150 CONTINUE
*
*     ISPEC = 11: infinity arithmetic can be trusted not to trap
*
*     ILAENV = 0
      ILAENV = 1
      IF( ILAENV.EQ.1 ) THEN
         ILAENV = IEEECK( **0**, 0.0, 1.0 )
      END IF
      RETURN
```

`INSTALL/tstiee.f`:

```
 1000 CONTINUE
*
*     ISPEC = 10: ieee NaN arithmetic can be trusted not to trap
*
      ILAENV = 1
      IF (ILAENV .EQ. 1) THEN
         ILAENV = IEEECK( **0**, 0.0, 1.0 )
      ENDIF
      RETURN
*
 1100 CONTINUE
*
*     ISPEC = 11: infinity arithmetic can be trusted not to trap
*
      ILAENV = 1
      IF (ILAENV .EQ. 1) THEN
         ILAENV = IEEECK( **1**, 0.0, 1.0 )
      ENDIF
      RETURN
```

**Comment by martin-frbg** [comment]
This was already raised in #435 - basically one of the two has its logic reversed but in both caller and callee (I think) -  so no problem in the end. IIRC the one in INSTALL is "only" used to create a standalone executable that is run before the actual library is built.

**Comment by youyu3** [comment]
Thanks for the clarification.  Now I see that in `INSTALL/tstiee.f` it's reversed in both the test driver and `ilaenv`.

**Comment by langou** [comment]
Hi @youyu3, thanks a lot, this is fixed in #498, I am closing this issue now. @langou
